docs/temporary_locations.md

```diff
@@ -108,4 +108,4 @@ In this case, in Orangelight, in order to display the status we match the holdin
 
 When a staff member moves an item from this temporary location to a permanent location the availability status will still display, because the holding id in the JS availability response matches the indexed holding id in the holdings_1display.
 
-When a staff member moves an item from a permanent location to this temporary location then there is a mismatch between the JS availability response and the not indexed yet holdings_1display["temp_location_code"]. The holdings_1display["temp_location_code"] is used from Orangelight in order to display the status in the specific temporary location "RES_SHARE$IN_RS_REQ". Finally this results in an empty availability status in the record page. This is fixed with the next incremental job.
\ No newline at end of file
+When a staff member moves an item from a permanent location to this temporary location then there is a mismatch between the JS availability response and the not indexed yet holdings_1display["temp_location_code"]. The holdings_1display["temp_location_code"] is used from Orangelight in order to display the status in the specific temporary location "RES_SHARE$IN_RS_REQ". Finally this results in an empty availability status in the record page. This is fixed with the next incremental job.
```

Comment:
Can you make [Orangelight] a link text and add in the link url the orangelight github repository url?